### DISCUSSION TITLE
How to plot graph with adruino uno

### DISCUSSION THREAD

**Discussion by solartech77** [rlogiacco/CircularBuffer#57] [General]
How do I plot a graph of voltage again time in real time? Getting data from my analog pin of the MCU.

**Comment by digamesystems** [comment]
Depends on the speed of your analog signal. For slow signals, where you want to look at them for debugging, you can simply Serial.println(AnalogSig); and use the Serial Plotter. Are you trying to use the CircularBuffer for your data? More detail would help folks answer your question.

**Reply by rlogiacco** [reply]
To double on @digamesystems answer, you definitely need to provide more info, but also allow me to highlight yours is a pretty generic question not much related to this library: you should post it on StackOverflow, which has a broader audience.

If you only need to plot for debug and you wish a graphical representation, I suggest you use Sloeber along with my PlotPlus library.

**Reply by solartech77** [reply]
Does print means it will  display on any screen attach to the mcu?
